prog25.rb

```diff
@@ -0,0 +1,72 @@
+# Q-25 Sales Tax Problem: You have to write a program in Ruby which would 
+# calculate the sales tax on the items according to the following conditions:
+# a. Flat 10 % sales tax is applicable on all items except for book, food and medicine
+# b. Additional Import Duty of 5 % is applicable on all imported items without any exceptions
+# The input to the program would be in the following format: 
+# Name of the product: Chocolates
+# Imported?: yes
+# Exempted from sales tax? Yes
+# Price: 120
+# Do you want to add more items to your list(y/n): y
+# Name of the product: Potato Chips
+# Imported?: no
+# Exempted from sales tax?: yes
+# Price: 100
+# Do you want to add more items to your list(y/n): y
+# Name of the product: Perfume
+# Imported?: yes
+# Exempted from sales tax? No
+# Price: 150
+# Do you want to add more items to your list(y/n): n
+# You have to generate a list that would display the list in a nice format 
+# and the grand total should be rounded to the nearest integer.
+
+
+class List
+  SALES_TAX = 0.1
+  IMPORT_DUTIES = 0.05
+  attr_accessor :hash
+  def initialize(hash)
+    @hash = hash
+  end
+  def total_price
+    final_list=Hash.new
+    @hash.each do | key, value |
+      if @hash[key][1] == "yes" && @hash[key][2] == "No"
+        @hash[key][0] = hash[key][0] + (@hash[key][0] * SALES_TAX) + (hash[key][0] * IMPORT_DUTIES)
+      elsif @hash[key][1] == "yes" && @hash[key][2] == "Yes"
```

Comment:
Instead of double checking here for case, you can always use a consistent case in your code. Example:
`a = gets.chomp.downcase`
`a == "yes"`
No need to check for upcase `Yes` then